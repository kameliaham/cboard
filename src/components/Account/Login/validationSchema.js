import * as yup from 'yup';

const validationSchema = yup.object().shape({
  email: yup
    .string()
    .email('Invalid email')
    .required('Required'),
  password: yup.string().required('Required')
});

export default validationSchema;
