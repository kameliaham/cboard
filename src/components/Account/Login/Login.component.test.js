import React from 'react';
import { shallow, mount } from 'enzyme';
import { shallowMatchSnapshot } from '../../../common/test_utils';
import { Login } from './Login.component';

jest.mock('./Login.messages', () => {
  return {
    login: {
      id: 'cboard.components.Login.login',
      defaultMessage: 'Login'
    },
    email: {
      id: 'cboard.components.Login.email',
      defaultMessage: 'Email'
    },
    password: {
      id: 'cboard.components.Login.password',
      defaultMessage: 'Password'
    },
    cancel: {
      id: 'cboard.components.Login.cancel',
      defaultMessage: 'Cancel'
    }
  };
});

const mockLoginfn = jest.fn();
const props = {
  isDialogOpen: false,
  onClose: jest.fn(),
  intl: {
    formatMessage: msg => msg
  },
  onResetPasswordClick: jest.fn()
};

describe('Login tests', () => {
  test('default renderer', () => {
    const wrapper = shallow(<Login {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
