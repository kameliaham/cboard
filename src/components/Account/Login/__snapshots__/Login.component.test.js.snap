// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login tests default renderer 1`] = `ShallowWrapper {}`;
